easycv/image.py

```diff
@@ -188,6 +197,18 @@ def compute(self, in_place=True):
             result = Image(self._pending(self._img)["image"], lazy=self._lazy)
             return result
 
+    def check_outs(self, outputs, transform):
```

Comment:
Codacy found an issue: [Method could be a function](https://app.codacy.com/gh/easycv/easycv/pullRequest?prid=6255518)